OasysGH/Components/Helpers/Output.cs

```diff
@@ -40,26 +51,36 @@ public static void SetItem(GH_OasysDropDownComponent owner, IGH_DataAccess DA, i
         owner.ExpireDownStream = false;
     }
 
-    public static void SetList<T>(GH_OasysDropDownComponent owner, IGH_DataAccess DA, int inputid, List<T> data)
+    public static void SetList<T>(GH_OasysDropDownComponent owner, IGH_DataAccess DA, int inputid, List<T> data) where T : IGH_Goo
     {
       DA.SetDataList(inputid, data);
 
       if (!owner.ExistingOutputsSerialized.ContainsKey(inputid))
       {
-        owner.ExistingOutputsSerialized.Add(inputid, new List<int>());
+        owner.ExistingOutputsSerialized.Add(inputid, new List<string>());
         owner.ExpireDownStream = true;
       }
 
       for (int i = 0; i < data.Count; i++)
       {
-        int outputsSerialized = 0;
+        string outputsSerialized = "";
         if (data[i].GetType() == typeof(GH_UnitNumber))
         {
           IQuantity quantity = ((GH_UnitNumber)(object)data[i]).Value;
-          outputsSerialized = JsonConvert.SerializeObject(quantity, converter).GetHashCode();
+          outputsSerialized = JsonConvert.SerializeObject(quantity, converter);
         }
         else
-          outputsSerialized = JsonConvert.SerializeObject(data, converter).GetHashCode();
+        {
```

Comment:
this looks like it could become a method that is called from two places (SetItem, SetList, etc.)

Reply:
https://github.com/arup-group/OasysGH/pull/10/commits/0c43d4d0e8a9a0a7e7202cd88fda4ba9fbe33fd6